lib/Dialect/CKKS/Conversions/CKKSToLWE/CKKSToLWE.cpp

```diff
@@ -0,0 +1,49 @@
+#include "lib/Dialect/CKKS/Conversions/CKKSToLWE/CKKSToLWE.h"
+
+#include <utility>
+
+#include "lib/Dialect/CKKS/IR/CKKSDialect.h"
+#include "lib/Dialect/CKKS/IR/CKKSOps.h"
+#include "lib/Dialect/LWE/IR/LWEDialect.h"
+#include "lib/Dialect/LWE/IR/LWEOps.h"
+#include "lib/Dialect/LWE/IR/LWEPatterns.h"
+#include "mlir/include/mlir/Dialect/Tensor/IR/Tensor.h"  // from @llvm-project
+#include "mlir/include/mlir/IR/MLIRContext.h"            // from @llvm-project
+#include "mlir/include/mlir/IR/PatternMatch.h"           // from @llvm-project
+#include "mlir/include/mlir/Support/LLVM.h"              // from @llvm-project
+#include "mlir/include/mlir/Support/LogicalResult.h"     // from @llvm-project
+#include "mlir/include/mlir/Transforms/GreedyPatternRewriteDriver.h"  // from @llvm-project
+
+namespace mlir::heir::ckks {
+
+#define GEN_PASS_DEF_CKKSTOLWE
+#include "lib/Dialect/CKKS/Conversions/CKKSToLWE/CKKSToLWE.h.inc"
+
+template <typename CKKSOp, typename LWEOp>
```

Comment:
This is a direct copy of the template class from BGV. Maybe put the common thing in `LWEPatterns.h` (or maybe a new RewriteUtils under lib/Utils?)

Reply:
I thought about putting it in ConversionUtils but didn't because it's not a conversion pattern, so I like the idea of RewriteUtils!